edit/workload/storage/index.vue

```diff
@@ -281,13 +299,22 @@ export default {
             />
           </div>
         </div>
-        <Mount :key="props.row.value.name" :container="container" :pod-spec="value" :name="props.row.value.name" :mode="mode" />
+
+        <!-- Mount point list to be mapped to volume -->
+        <Mount
+          :container="container"
```

Comment:
Removing the Mount key causes issues when deleting a volume. Try:

1. Add a volume
2. Define a mount point
3. Add a second volume
4. Define a mount point for that volume
5. Delete the first volume

Observe that the second (now only) volume lists both mount points

Reply:
@mantis-toboggan-md Should this issue be resolved in this PR or should it be tackled via a separate (github) issue?

Reply:
It'd certainly be preferable not to knowingly introduce a bug while fixing another, but I've just done a little investigation myself and it looks like it wont be a quick fix to resolve both #5317 and what I've described. Given that the former is more obnoxious and our deadline approaches, I think it's probably fine to track the latter separately and fix it in 2.6.6.

Reply:
I can try to generate an identifier which is not paired with the name.

Reply:
It seems like before it was always removing every mount, while now is not, but now I got some race conditions while creating partial copies or references of the original value/container/volume.
Working on it.